### DISCUSSION TITLE
Remove the :negative_squared_crossed_mark: from the dungeon emote list

### DISCUSSION THREAD

**Discussion by Sirfirelord69** [DiscordDungeons/Bugs#1115] [Ideas]
As far as I can see it serves no purpose, and it just causes more emote lag

**Comment by mount2010** [comment]
I mean since we are going to redesign every reaction interface for buttons soon it's probably not worth doing this now

**Comment by Willard21** [comment]
Remove it from the button list then :P

**Comment by FelixArgyle** [comment]
Replace it with a new button that checks your Dungeon Status (instead of completely removing it)
